RealmSwift/Tests/PerformanceTests.swift

```diff
@@ -0,0 +1,462 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import XCTest
+import RealmSwift
+
+#if !DEBUG
+
+private func createStringObjects(factor: Int) -> Realm {
+    let realm = Realm(inMemoryIdentifier: factor.description)
+    realm.write {
+        for _ in 0..<(1000 * factor) {
+            realm.create(SwiftStringObject.self, value: ["a"])
+            realm.create(SwiftStringObject.self, value: ["b"])
+        }
+    }
+    return realm
+}
+private var smallRealm: Realm!
+private var mediumRealm: Realm!
+private var largeRealm: Realm!
+
+class SwiftPerformanceTests: TestCase {
+    override class func setUp() {
```

Comment:
The fact that this needs to be explained here rather than being clear from the code/comments says to me that we either need to change this or document it.

Is there a reason we can't simply create the Realm outside of `measureBlock`/`inMeasureBlock`?